### ISSUE TITLE
Add filament monitor data to OM including for CAN-connected boards

### ISSUE THREAD

**Issue by T3P3** [Duet3D/RepRapFirmware#756]
This is the less frequently refreshed information such as configuration, AGC reading, calibration etc.

There is a separate issue for the semi-realtime position data:

https://github.com/Duet3D/RepRapFirmware/issues/702

**Comment by chrishamm** [comment]
The calibration part is implemented on 3.7-ch (RRF + Duet3Expansion): `sensors.filamentMonitors[].calibrated` is now populated for CAN-connected monitors. Pulsed monitors sent no live data at all, but now they fill in the same fields.

Still open: agc/mag and brightness/shutter. There are 11 spare bits in `FilamentMonitorDataV2`, so the pair doesn't fit as two 8-bit fields.

**Comment by T3P3** [comment]
brightness/shutter is not needed. I propose we depreciate support for the laserFM in 3.7

"agc" is much more useful than mag so potentially only report that?

**Comment by chrishamm** [comment]
Yes, I agree, agc could be useful in the OM. To be shared with shutter in the CAN message.

**Comment by chrishamm** [comment]
This is tested and working using a remotely connected MFM (via 3HC).
